.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Description
+
+Fixes #<ISSUE-NUMBER>
+
+Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.
+
+## Checklist
+- [ ] I have followed [Contributing Guidelines from CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md)
```

Comment:
nit: I think in addition to the Contributing guidelines checkbox, mention explicitly the style guide here something like including following recommendations from the style guide https://googlecloudplatform.github.io/samples-style-guide/

Reply:
Updated. PTAL.